test-e2e/basic.test.ts

```diff
@@ -0,0 +1,27 @@
+import { vscodeTest } from '@hiogawa/vscode-e2e/vitest'
+import { beforeEach } from 'vitest'
+
+beforeEach(({ task }) => {
+  task.meta.vscodeExtensionPath = '.'
+  task.meta.vscodeWorkspacePath = './samples/e2e'
+  task.meta.vscodeTrace = 'on'
```

Comment:
Maybe just use `vscodeTest.extend`?

```ts
const test = vscodeTest.extend({
  extensionPath: '.',
  workspacePath: './samples/e2e'
  trace: 'on'
})
```

And inside `_launch` you can just access them:

```ts
const vscodeTest = test.extend({
  _launch: async ({ extensionPath, workspacePath }, use) => {
    const _launch = await launch({
      extensionPath,
      workspacePath,
    });
    await use(_launch);
    await _launch.app.close();
  },
})
```

I assume they are required 🤔

Reply:
I agree that using `task.meta` doesn't feel convenient, but If API requires users to do one more `test.extend` themselves, then I would simply expose something like `createVscodeTest` instead of exposing "partial" fixture `vscodeTest`.
This is a prototype for that idea https://github.com/vitest-dev/vscode/pull/276/commits/dd5dba56001632c0029b0d2909bd99b8ee57a5fc

Reply:
I think it's absolutely fine to call extend several times, this is the recommended way in playwright for example. The basic test function is already called with playwright to define "page" and other fixtures for example

Reply:
Yeah, I understand it's a valid API for users to do `test.extend`, but while iterating on the API, I just started to feel I'm over-engineering for what would be much more direct and simpler.

For now, I'll take back my package and I would be more comfortable to just copy minimal implementation here.

The one interesting thing Playwright's fixture can do is that, since they have fixture options https://playwright.dev/docs/next/test-fixtures#fixtures-options, they can pick up certain configuration from their config file to be used as their default fixture customization.

If Vitest would have this feature, then that could be another potential API to configure `extensionPath`.